packages/embed/src/factories/create-popover/create-popover.ts

```diff
@@ -97,6 +97,48 @@ const buildTooltip = (message: string, hide: () => void) => {
   return container
 }
 
+const LOCAL_STORAGE_NOTIFICATION_DATA = 'tfNotificationData'
+
+type LocalStorageCUIData = {
+  formId: string
+  expireDate: number
+}
+const today = new Date().getTime()
+
+const cuiLocaslStorageDate = typeof window !== 'undefined' && localStorage.getItem(LOCAL_STORAGE_NOTIFICATION_DATA)
+
+const getLocalStorageData = () => cuiLocaslStorageDate && JSON.parse(cuiLocaslStorageDate)
+
+function saveToLocalstorage(data: LocalStorageCUIData) {
+  if (!data) return null
+  typeof window != 'undefined' && localStorage.setItem(LOCAL_STORAGE_NOTIFICATION_DATA, JSON.stringify(data))
+}
```

Comment:
Please capitalize the word `storage`:
```suggestion
function saveToLocalStorage(data: LocalStorageCUIData) {
  if (data && typeof window != 'undefined' ) {
    localStorage.setItem(LOCAL_STORAGE_NOTIFICATION_DATA, JSON.stringify(data))
  }
}
```

Reply:
Also - is the `typeof window != 'undefined'` check necessary? Is the code ever executed server-side? If this is just for test we should mock it.

Reply:
I am mocking localStorage in the tests